corehq/apps/userreports/reports/filters/values.py

```diff
@@ -128,35 +133,129 @@ def to_sql_values(self):
         }
 
 
+def get_dyn_range_filter(date_range_slug):
```

Comment:
i think it'd be helpful to name this (and the class) something that makes it obvious that it's a dynamic date range and not something more general